connectivity/mbedtls/tools/importer/adjust-config.sh

```diff
@@ -117,6 +117,9 @@ conf unset MBEDTLS_SSL_TRUNCATED_HMAC
 
 conf unset MBEDTLS_PLATFORM_TIME_TYPE_MACRO
 
+# potentially save flash space by not enabling self-tests by default
```

Comment:
The commit message for "Move Mbed TLS self tests to a separate configuration" should be updated, too, not just this comment. It still says "there is no point..."

Reply:
Done